ocs_ci/helpers/helpers.py

```diff
@@ -3334,3 +3334,22 @@ def get_failure_domain():
 
     storage_cluster_obj = get_storage_cluster()
     return storage_cluster_obj.data["items"][0]["status"]["failureDomain"]
+
+
+def get_pod_logs(
+    pod_name, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE, all_containers=True
+):
+    """
+    Get pod logs
+
+    Args:
+        pod_name (str): The name of the pod
+        namespace (str): The name of the namespace to use
+        all_containers (bool): fetch logs from all containers of the resource
```

Comment:
Please make it more informative like what if False is passed.

Reply:
I deleted this function